app/controllers/api/v1/classifications_controller.rb

```diff
@@ -34,6 +34,14 @@ def project
     index
   end
 
+  def offset
+    @controlled_resources = Classification.where("project_id = ? AND id > ?", params[:project_id], params[:last_id])
+    .includes(:project)
+    .includes(:user)
+    render json_api: serializer.page(params, controlled_resources, context),
```

Comment:
I didn't change this to use a scope because of the fact that project and classification ids are required as params. Not sure how to provide these to a scope...could use a lambda in the scope def and pass it some params, probably. Would that be worth doing?